### PR TITLE
#151 hexo & #152 joplin

### PR DESCRIPTION
**Pull request by pramit-d**
added #151 & #152 to OSSP-master.js
created 151-hexo-Versiondetails.json & 152-joplin-Versiondetails.json

### CODE REVIEW

**Comment by asa1997** [comment]
@pramit-d Also mention the issue number for joplin.

**Comment by pramit-d** [comment]
> @pramit-d Also mention the issue number for joplin.

@asa1997 done

**Review by asa1997 — APPROVED** [review]
LGTM

File: bes_theme/assets/data/version_details/151-hexo-Versiondetails.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+[
+    {
+        "version": "6.2.0",
+        "release_date": "11-May-2022",
+        "criticality_score": "Not Available",
```
Comment (sudhirverma) [inline bes_theme/assets/data/version_details/151-hexo-Versiondetails.json:5]: criticality_score and scorecard details are not available?

Comment (pramit-d) [reply bes_theme/assets/data/version_details/151-hexo-Versiondetails.json:5]: > criticality_score and scorecard details are not available?

I don't know how to do it, today I will to.

**Pull request merged**